src/sbt-test/sbt-scalafix/basic/project/plugins.sbt

```diff
@@ -1,3 +1,6 @@
-resolvers += Resolver.sonatypeRepo("public")
+resolvers += MavenRepository(
+  "sonatype-central-maven-snapshots",
+  "https://central.sonatype.com/repository/maven-snapshots/"
+)
```

Comment:
verbose explicit definition needed to support older yet tested sbt 1.x version that predate the introduction of `Resolver.sonatypeCentralSnapshots`